app/controller.js

```diff
@@ -856,6 +856,12 @@ class MainController {
 
         region.element.classList.add("selected-region");
 
+        region.data.words.forEach((w) => {
+            if (!w.uuid) {
```

Comment:
I do not understand why do we manipulate data on 'selectRegion' function.
Why not to execute this code while loading the data and every time a new word appears

Reply:
There is just in optimization purposes - i.e. for demo file we need 1497 uuids on a loading, I think creating uuids on demand it's a better way